.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Revert this file. It's the file created by RubyMine IDE.

Reply:
Need your help.

Reply:
Add It to `.gitignore` file.

Reply:
You need to remove It too.